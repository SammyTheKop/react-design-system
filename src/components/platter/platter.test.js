// react
import React from "react";

// component
import Platter from "./platter";

// jest renderer
import renderer from "react-test-renderer";

// testing library react
import "@testing-library/jest-dom/extend-expect";

// Case 1: Button Renders properly or not
it("Case 1: Button Component Renders Correctly", () => {
  const platterComponentTree = renderer
    .create(<Platter headerName="Platter Header">Platter Contents</Platter>)
    .toJSON();
  expect(platterComponentTree).toMatchSnapshot();
});
