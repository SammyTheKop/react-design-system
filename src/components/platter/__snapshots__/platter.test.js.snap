// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Case 1: Button Component Renders Correctly 1`] = `
Array [
  <div
    className="platter-container"
  >
    <div
      className="platter-container-header"
    >
      Platter Header
    </div>
    <div
      className="platter-container-contents"
    >
      Platter Contents
    </div>
  </div>,
  <style
    jsx={true}
  >
    
          /* Styles for the Platter Component */
          .platter-container {
            background: #e2e4e6;
            border-radius: 6px;
            height: 100%;
          }

          .platter-container-header {
            display: flex;
            justify-content: center;
            align-items: center;
            font-weight: 700;
            margin-bottom: 16px;
          }

          .platter-container-contents {
            height: 90%;
            background-color: #c5c5c5;
            border-radius: 6px;
          }
        
  </style>,
]
`;
